Makefile

```diff
@@ -1,35 +1,25 @@
 
-.PHONY: build
-run: docker-build
-
+# Run API service
 .PHONY: run
-run: docker-run
-
-.PHONY: test
-test: docker-test
-
-# Build all executables
-.PHONY: docker-run
-docker-run: docker-build
+run: build
 	docker run -t -i --rm onflow.org/api-service
 
 # Run build/test/run debug console
-.PHONY: docker-debug

```

Comment:
It seems the convention in flow-go (and more generally) is to use the docker prefix where docker is being invoked in the make command. Ok if we add those back in?

Reply:
We can create separate local and docker ones and default to docker for now. It is easier to install prerequisites specifying a single docker build environment, especially since that one also runs on the local machine.